tests/plugins/application/test_serverless_app_plugin.py

```diff
@@ -220,6 +220,15 @@ def test_resolve_intrinsics(self):
 
         self.assertEqual("value1", output)
 
+    def test_resolve_invalid_mapping(self):
```

Comment:
could you please add some unit test cases for invalid templates like:
- mappings = {"MapA": "value"}
- mappings = {"MapA": None}